securedrop/static/css/source.css

```diff
@@ -392,6 +392,32 @@ button.center {
   font-size: 18pt;
 }
 
+input#show {
+  display:none;
+}
+#content {
+    display: block;
+    -webkit-transition: opacity 1s ease-out;
```

Comment:
This is probably unnecessary, all major web browsers have supported unprefixed `transition:` since 2013: http://www.web-plus-plus.com/Articles/css-transition-moz-and-webkit-vs-css3. Additionally, we tend to focus on Tor Browser/Firefox, since that is the strongly recommended client to use when accessing SecureDrop sites, and `-webkit-transition` will have no effect there.